cmd/server/server.go

```diff
@@ -50,6 +50,7 @@ type serverArgs struct {
 	// mixer manager args
 	serviceConfigFile      string
 	globalConfigFile       string
+	kubeConfig             string
```

Comment:
I really don't like to see k8s stuff creeping into the core parts of our design.

Can we encapsulate all details of config maps into a completely separate binary that communicates with mixer's generic API? It'd be a bit more complicated, but it would maintain isolation between our core and k8s.